packages/react-components/src/components/utils/timegraph-container-component.tsx

```diff
@@ -33,14 +33,17 @@ export class ReactTimeGraphContainer extends React.Component<ReactTimeGraphConta
     }
 
     componentWillUnmount(): void {
-        if (this.container) {
-            this.container.destroy();
-        }
         if (this._resizeHandler) {
             this.props.removeWidgetResizeHandler(this._resizeHandler);
         }
     }
 
+    destroyContainer(): void {
```

Comment:
I get the following sonar lint error:

Remove this property or method or refactor "ReactTimeGraphContainer", as "destroyContainer" is not used inside component bodysonarlint(typescript:S6441)

see https://rules.sonarsource.com/javascript/RSPEC-6441/

It sounds worth fixing.

Reply:
I had to do this on purpose for the following reasons:
- if the number of row models is 0, the container gets destroyed, which then disposes the layers that were added to it. This becomes a problem, because if the Output_Data_Changed signal is received in the timegraph output component and the row model size is greater than 0, updateChart would throw a bunch of errors because the layers were disposed. So, the way i fixed this for now is to destroy the container when the timegraph-output-component is unmounted.